### PR TITLE
Updates for community and certified 1.1.0 operator

### LINKED ISSUE #1710: RedHat Operator Updates for 1.1.0
Further updates for operator to work within RedHat are needed to allow certified operator usage.

  * further related images
     * storage_initializer
     * prepackaged servers
     * alibi explainer
     * request logger
     * alibi-detect server

### PR DESCRIPTION
**Pull request by cliveseldon**
Fixes #1710

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **cliveseldon**

If they are not already assigned, you can assign the PR to them by writing `/assign @cliveseldon` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Sun Apr 19 09:26:47 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=1

**Comment by seldondev** [comment]
Sun Apr 19 09:26:48 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=2

**Comment by seldondev** [comment]
Mon Apr 20 19:34:34 UTC 2020
The logs for [lint] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/4.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=4

**Comment by seldondev** [comment]
Mon Apr 20 19:34:42 UTC 2020
The logs for [pr-build] [3] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/3.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=3

**Comment by seldondev** [comment]
Tue Apr 21 16:29:24 UTC 2020
The logs for [pr-build] [5] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/5.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=5

**Comment by seldondev** [comment]
@cliveseldon: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 278e954d01c69375b72924b4339d88f6d131b1fe | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Wed Apr 22 19:05:57 UTC 2020
The logs for [lint] [8] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/8.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=8

**Comment by seldondev** [comment]
Wed Apr 22 19:06:03 UTC 2020
The logs for [pr-build] [7] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/7.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=7

**Comment by seldondev** [comment]
Thu Apr 23 13:54:04 UTC 2020
The logs for [lint] [10] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/10.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=10

**Comment by seldondev** [comment]
Thu Apr 23 13:54:14 UTC 2020
The logs for [pr-build] [9] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/9.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=9

**Comment by seldondev** [comment]
@cliveseldon: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | a94129ec7176ef8519d7e57b6c7a6aede321eb13 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Thu Apr 23 15:33:30 UTC 2020
The logs for [pr-build] [11] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/11.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=11

**Comment by seldondev** [comment]
Thu Apr 23 15:33:33 UTC 2020
The logs for [lint] [12] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/12.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=12

**Comment by seldondev** [comment]
Thu Apr 23 17:20:41 UTC 2020
The logs for [pr-build] [13] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/13.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=13

**Comment by seldondev** [comment]
Thu Apr 23 17:20:41 UTC 2020
The logs for [lint] [14] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/14.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=14

File: version_release.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+1.1.0
```
Comment (axsaucedo) [inline version_release.txt:1]: So we're now gonna have a version.txt with SNAPSHOT and one which is version_release.txt with version release? Would it not be better to just have one and create a redhat release branch with the release version? This is what we do for all dated (and full) releases

File: operator/seldon-operator/bundle/certified/1.1.0/seldon-operator.v1.1.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,527 @@
...
+          "spec": {
+            "name": "test-deployment",
+            "predictors": [
+              {
+                "componentSpecs": [
+                  {
+                    "spec": {
+                      "containers": [
+                        {
+                          "image": "registry.connect.redhat.com/seldonio/mock-classifier@sha256:482ee477c344badcaa80e850f4339db41957f9c2396ae24f9e398b67bd5c184e",
```
Comment (axsaucedo) [inline operator/seldon-operator/bundle/certified/1.1.0/seldon-operator.v1.1.0.clusterserviceversion.yaml:22]: Oh wow does it have to have a hash for tag? Can it not have the one you did above with tag 1.1.0?

Comment (cliveseldon) [reply operator/seldon-operator/bundle/certified/1.1.0/seldon-operator.v1.1.0.clusterserviceversion.yaml:22]: not inside the example olm not unfortunately

File: version_release.txt (hunk 1 above)
Comment (cliveseldon) [reply version_release.txt:1]: yes that might be better to be able to just use the release version. Lets do it in the release issue: #1732

File: operator/seldon-operator/bundle/certified/0.1.5/seldondeployments.machinelearning.seldon.io.crd.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    kind: SeldonDeployment
+    listKind: SeldonDeploymentList
+    plural: seldondeployments
+    shortNames:
+    - sdep
+    singular: seldondeployment
+  scope: Namespaced
+  subresources:
+    status: {}
+  version: v1alpha2
```
[the remark this answers is not in the document]
Comment (cliveseldon) [reply operator/seldon-operator/bundle/certified/0.1.5/seldondeployments.machinelearning.seldon.io.crd.yaml:17]: 0.1.5 is old one

File: components/alibi-detect-server/Dockerfile.redhat [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      description="The Alibi Detect Server provides outlier, drift and adversarial detection services for Seldon Core"
+
+ADD requirements_server.txt .
+
+RUN pip install pip -U
+
+RUN pip install -r requirements_server.txt
+
+# Fix cloudevents bug: https://github.com/cloudevents/sdk-python/issues/24
+RUN git clone --branch 24-extensions https://github.com/ryandawsonuk/sdk-python.git && \
```
[the remark this answers is not in the document]
Comment (cliveseldon) [reply components/alibi-detect-server/Dockerfile.redhat:16]: Well @ryandawsonuk did a fix for cloudevents but its not been accepted for some reason.

File: operator/seldon-operator/bundle/certified/1.1.0/seldon-operator.v1.1.0.clusterserviceversion.yaml (hunk 2 above)
Comment (axsaucedo) [reply operator/seldon-operator/bundle/certified/1.1.0/seldon-operator.v1.1.0.clusterserviceversion.yaml:22]: Oh wow, fair enough!

File: version_release.txt (hunk 1 above)
Comment (axsaucedo) [reply version_release.txt:1]: Yeah sounds good!

File: operator/seldon-operator/bundle/certified/0.1.5/seldondeployments.machinelearning.seldon.io.crd.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (axsaucedo) [reply operator/seldon-operator/bundle/certified/0.1.5/seldondeployments.machinelearning.seldon.io.crd.yaml:17]: Got it!

File: components/alibi-detect-server/Dockerfile.redhat (hunk 4 above)
[the remark this answers is not in the document]
Comment (axsaucedo) [reply components/alibi-detect-server/Dockerfile.redhat:16]: Oh right, ok that may take a while

**Comment by seldondev** [comment]
@cliveseldon: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint | b231abfd50b8e1ccab731b009ad13a1d18de68bd | [link]() | `/test lint`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@cliveseldon: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint | b231abfd50b8e1ccab731b009ad13a1d18de68bd | [link]() | `/test lint`
pr-build | b231abfd50b8e1ccab731b009ad13a1d18de68bd | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Tue Apr 28 08:13:47 UTC 2020
The logs for [lint] [18] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/18.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=18

**Comment by seldondev** [comment]
Tue Apr 28 08:13:52 UTC 2020
The logs for [pr-build] [17] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/17.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=17

**Comment by seldondev** [comment]
Tue Apr 28 08:15:44 UTC 2020
The logs for [pr-build] [19] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/19.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=19

**Comment by seldondev** [comment]
Tue Apr 28 08:15:44 UTC 2020
The logs for [lint] [20] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/20.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=20

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/SeldonIO/seldon-core/pull/1711#issuecomment-621114849" title="Approved">axsaucedo</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [axsaucedo]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Wed Apr 29 10:28:09 UTC 2020
The logs for [lint] [22] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/22.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=22

**Comment by seldondev** [comment]
Wed Apr 29 10:28:20 UTC 2020
The logs for [pr-build] [21] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1711/21.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1711 --build=21

**Comment by seldondev** [comment]
@cliveseldon: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | fe3b1aa58e339ea3b69c0a03c3cc5b8b17522939 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Pull request merged**